Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule.py

```diff
@@ -330,8 +330,8 @@ def init_roots(self):
             # try TAXII 2.1
             self.set_api_root()
         # (TAXIIServiceException, HTTPError) should suffice, but sometimes it raises another type of HTTPError
-        except Exception as e:
-            if "406 Client Error" not in str(e) and "version=2.0" not in str(e):
+        except HTTPError as e:
```

Comment:
why did you narrow it to HTTPError, could it be received as a different type of exception? also it needs to be added to the import section, as seen in the failing checks.

Reply:
From the comment `but sometimes it raises another type of HTTPError` and from the error messages I thought, that it can only raise HTTPErrors. Nevertheless, I will leave checks for plain Exception.